app/controllers/users_controller.rb

```diff
@@ -2,6 +2,7 @@ class UsersController < ApplicationController
   layout 'landing', :only => [:new, :create]
   load_and_authorize_resource only: [:edit, :update]
   skip_before_action :authenticate!
+  attr_accessor :redirect_url
```

Comment:
attr_accessor in a controller this is something i never saw... Why did you add this it's not even something you need...